### ISSUE TITLE
Implement StatsBase.coefnames for LMs

### ISSUE THREAD

**Issue by kescobo** [JuliaStats/GLM.jl#235]
Example from the docs:

```
using DataFrames, CategoricalArrays, GLM

dobson = DataFrame(Counts    = [18.,17,15,20,10,20,25,13,12],
                                 Outcome   = categorical([1,2,3,1,2,3,1,2,3]),
                                 Treatment = categorical([1,1,1,2,2,2,3,3,3]))

gm1 = fit(GeneralizedLinearModel, @formula(Counts ~ Outcome + Treatment), dobson,
 Poisson())

coefnames(gm1)
```

Gives the error:

```
ERROR: coefnames is not defined for StatsModels.DataFrameRegressionModel{GLM.GeneralizedLinearModel{GLM.GlmResp{Array{Float64,1},Distributions.Poisson{Float64},GLM.LogLink},GLM.DensePredChol{Float64,Base.LinAlg.Cholesky{Float64,Array{Float64,2}}}},Array{Float64,2}}.
```

With some direction, I'm happy to do a PR, but my naive attempt:

```
import StatsBase.coefnames

coefnames(obj::LinearModel) = coeftable(obj).rownms
```

didn't work. I'm guessing I just need to find the right abstract type that encompasses everything?

**Comment by nalimilan** [comment]
The error happens when calling `coefnames` on `DataFrameRegressionModel`. Maybe we just need to add that function to the list of delegated methods in StatsModels.jl (see src/statsmodel.jl).

**Comment by kleinschmidt** [comment]
To expand on @nalimilan's point: the reason that didn't work is because you actually get a wrapped model of type `DataFrameRegressionModel` instead of `LinearModel`.  So the `coefnames` method needs to be added for that type.

**Comment by kleinschmidt** [comment]
There's a StatsModels release tagged which should fix this (v0.2.5) JuliaLang/METADATA.jl#14766.
